sam_bot_description/launch/display.launch.py

```diff
@@ -1,60 +1,107 @@
-import launch
-from launch.substitutions import Command, LaunchConfiguration
-import launch_ros
 import os
 
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, ExecuteProcess, IncludeLaunchDescription
+from launch.conditions import IfCondition, UnlessCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import Command, LaunchConfiguration
+from launch_ros.actions import Node
+from ros_gz_bridge.actions import RosGzBridge
+from ros_gz_sim.actions import GzServer
+
+
 def generate_launch_description():
-    pkg_share = launch_ros.substitutions.FindPackageShare(package='sam_bot_description').find('sam_bot_description')
+    pkg_share = get_package_share_directory('sam_bot_description')
+    ros_gz_sim_share = get_package_share_directory('ros_gz_sim')
+    gz_spawn_model_launch_source = os.path.join(ros_gz_sim_share, "launch", "gz_spawn_model.launch.py")
     default_model_path = os.path.join(pkg_share, 'src/description/sam_bot_description.urdf')
     default_rviz_config_path = os.path.join(pkg_share, 'rviz/urdf_config.rviz')
-    world_path=os.path.join(pkg_share, 'world/my_world.sdf')
-    
-    robot_state_publisher_node = launch_ros.actions.Node(
+    world_path = os.path.join(pkg_share, 'world/my_world.sdf')
```

Comment:
```suggestion
    world_path = os.path.join(pkg_share, 'world', 'my_world.sdf')
```

Is the more correct way of doing this, for all around here as well :-) 

I know it was wrong before :wink: